frontend/src/pages/EditUserPage.tsx

```diff
@@ -88,9 +88,9 @@ const EditUserPage = (props: IUserDetailParams) => {
 
   checkRoles =
     user.roles.length > 0 ? (
-      <Select field="role" placeholder={user.roles.find(x => x).name} options={[]} />
+      <Select field="role" placeholder={user.roles[0].name} options={[]} />
```

Comment:
Did you want this hardcoded to only one role?

Reply:
For now since we only allow the user to have one role or agency (on the FE) I had set it up to display one role. I originally had it so it would just push the new role/agency onto the existing ones but since the front end only supports one at a time I changed it